bench/Main.hs

```diff
@@ -0,0 +1,282 @@
+
```

Comment:
This file is a rename of `ghcide/bench/hist/Main.hs`, but for some reason Github doesn't show it